<launch>


<node pkg="fisheye_model" type="stereoCropper" name="stereoRectify" output="screen">
    <remap from="/left_image" to="/djiros/image_left"/>
    <remap from="/right_image" to="/djiros/image_right"/>
    <param name="imgL" type="string" value="/stereo/left"/>
    <param name="imgR" type="string" value="/stereo/right"/>
    <param name="imgFInfo" type="string" value="up/camera_info"/>
    <param name="camera_model_left" type="string" value="$(find fisheye_model)/config/left_fisheye_camera_calib.yaml"/>
    <param name="camera_model_right" type="string" value="$(find fisheye_model)/config/right_fisheye_camera_calib.yaml"/>
    <param name="cropper_file" type="string" value="$(find fisheye_model)/config/stereo.yaml"/>
    <param name="is_image_show" type="bool" value="true"/>
    <param name="angle_row" type="double" value="70.0"/>
    <param name="angle_col" type="double" value="55.0"/>
    <param name="max_image_size" type="int" value="320"/>
 </node>

</launch>
